Doc/c-api/init.rst

```diff
@@ -1279,6 +1279,21 @@ with sub-interpreters:
       Hangs the current thread, rather than terminating it, if called while the
       interpreter is finalizing.
 
+
+.. c:function:: int PyGILState_EnsureOrFail(PyGILState_STATE *state)
+
+   Similar to :c:func:`PyGILState_Ensure`, except that it returns with a status
+   code even in the case of failure. Specifically, it returns ``0`` when the
+   operation succeeded, and ``-1`` otherwise. In contrast to
+   :c:func:`PyGILState_Ensure`, *state* is an argument.
+
+   In the case of failure, it is *unsafe* to use the Python API following the
+   call. Releasing the obtained *state* via :c:func:`PyGILState_Release` should
```

Comment:
```suggestion
   call. Releasing the obtained *state* via :c:func:`PyGILState_Release` must
```

Reply:
I applied this suggestion to https://github.com/python/cpython/pull/130012 since I closed this PR.